### PR TITLE
Cameroon update to informant type value

### PR DESCRIPTION
**Pull request by wilrona**

### CODE REVIEW

File: packages/client/src/forms/configuration/default/birth.ts [hunk 1]
```diff
@@ -577,6 +577,10 @@ export const birthRegisterForms: ISerializedForm = {
             fieldName: 'informantType',
             operation: 'informantTypeTransformer'
           },
+          {
+            fieldName: 'informantTypeValue',
+            operation: 'informantTypeTransformerValue'
```
Comment (tahmidrahman-dsi) [inline packages/client/src/forms/configuration/default/birth.ts:582]: @wilrona Don't we already have `informantTypeTransformer` ?

Comment (wilrona) [reply packages/client/src/forms/configuration/default/birth.ts:582]: We have some but it returns the informantType information already translated into the current language. As a result, I no longer have the MOTHER or FATHER key needed to make the conditions on the part of the certificate concerned.

Comment (wilrona) [reply packages/client/src/forms/configuration/default/birth.ts:582]: That's why I just duplicated what existed to get these keys.

Comment (tahmidrahman-dsi) [reply packages/client/src/forms/configuration/default/birth.ts:582]: Ok, looks good 👍🏼

**Pull request merged**